index.js

```diff
@@ -1,2 +1,6 @@
+const config = require('pelias-config');
 
-module.exports = require('./src/sink');
\ No newline at end of file
+module.exports = (function() {
+  require('./src/configValidation').validate(config.generate());
```

Comment:
tricky. does it work if you move line 4 out on its own? If so you can get rid of the [IIFE](http://benalman.com/news/2010/11/immediately-invoked-function-expression/) which would be nice.